src/apiCalls.js

```diff
@@ -1,12 +1,17 @@
 import axios from 'axios'
+const rootUrl = 'https://som-timer-be.herokuapp.com/api'
 
 export const getSettings = () => {
-  return axios.get('https://som-timer-be.herokuapp.com/api/timers/1')
+  return axios.get(`${rootUrl}/timers/1`)
 }
 
 export const updateSettings = (settings) => {
-  return axios.put('https://som-timer-be.herokuapp.com/api/timers/1', {
+  return axios.put(`${rootUrl}/timers/1`, {
     work_interval: settings.workInterval,
     rest_interval: settings.breakInterval
   })
+}
+
+export const getRandomContent = (duration, category) => {
```

Comment:
Glad this worked out so easily once we changed the BE to take Query params over a request body.

Reply:
Yes, thank you so much for bee-ing so responsive and helpful all the time (to you and all the bees!)